interactive_engine/executor/ir/integrated/src/bin/start_rpc_server_k8s.rs

```diff
@@ -0,0 +1,138 @@
+//
+//! Copyright 2022 Alibaba Group Holding Limited.
+//!
+//! Licensed under the Apache License, Version 2.0 (the "License");
+//! you may not use this file except in compliance with the License.
+//! You may obtain a copy of the License at
+//!
+//! http://www.apache.org/licenses/LICENSE-2.0
+//!
+//! Unless required by applicable law or agreed to in writing, software
+//! distributed under the License is distributed on an "AS IS" BASIS,
+//! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//! See the License for the specific language governing permissions and
+//! limitations under the License.
+
+use std::env;
+
+use log::info;
+use runtime_integration::{InitializeJobAssembly, QueryExpGraph};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    pegasus_common::logs::init_log();
+    let id = get_id();
+    let servers_size = get_servers_size();
+    let dns_name = get_dns_name();
+    let engine_port = get_engine_port();
+    let rpc_port = get_rpc_port();
+    let server_config_string = generate_server_config_string(id, servers_size, &dns_name, engine_port);
```

Comment:
Can we implement a structure with some generic type to handle start_rpc_server standalone and with k8s? Can also do so for k8s.rs and standalone.rs.